Mathlib/MeasureTheory/MeasurableSpace/Basic.lean

```diff
@@ -1615,7 +1615,7 @@ def piCongrLeft (f : δ ≃ δ') : (∀ b, π (f b)) ≃ᵐ ∀ a, π a := by
   exact fun i => measurable_pi_apply (f i)
 
 theorem piCongrLeft_eq (f : δ ≃ δ') :
-    MeasurableEquiv.piCongrLeft π f = f.piCongrLeft π := by rfl
+    ⇑MeasurableEquiv.piCongrLeft π f = f.piCongrLeft π := by rfl
```

Comment:
Is this a stylistic thing, or was this somehow broken by the changes above?

Reply:
The lemmas were equalities of `Equiv _ _`, but I want them to be equalities of functions. This was a mistake in the old version.
The coercion from `MeasurableEquiv` to `Equiv` (in the old LHS) just doesn't show up often, but the coercion to functions does.